Dịch "chapter_preliminaries/pandas_vn" - Phần 01 (#448)
aivivn/d2l-vn: chapter_preliminaries/pandas_vn.md

We will cover more data preprocessing techniques in later chapters.
-->

> Trước tới nay chúng ta đã đề cập tới rất nhiều kỹ thuật thao tác dữ liệu được lưu trong dạng `ndarray`.

Reviewer: Từ "thao tác" (manipulation) hay quá. Mình sẽ đổi tiêu đề cho mục "Data Manipulation" thành "Thao tác với dữ liệu"

Nhưng để áp dụng học sâu vào giải quyết các vấn đề thực tế, ta thường phải bắt đầu bằng việc xử lý dữ liệu thô, chứ không phải luôn có ngay dữ liệu ngăn nắp đã chuẩn bị sẵn trong định dạng `ndarray`
Trong số các công cụ phân tích dữ liệu phổ biến của Python, gói `pandas` hay được sử dụng nhiều.
Giống nhiều gói khác trong hệ sinh thái Python, `pandas` có thể làm việc cùng định dạng `ndarray`.
Vì vậy, chúng ta sẽ đi nhanh qua các bước để tiền xử lý dữ liệu thô bằng `pandas` rồi đổi chúng sang dạng `ndarray`.

> Sau đó ta sẽ bao quát nhiều kỹ thuật xử lý dữ liệu hơn trong các chương sau.

Reviewer 2: ```suggestion
Sau đó ta sẽ bao quát nhiều kỹ thuật tiền xử lý dữ liệu hơn trong các chương sau.
```

The comment `# Saved in the d2l package for later use` is a special mark where the following function, class, or import statements are also saved in the `d2l` package so that we can directly invoke `d2l.mkdir_if_not_exist()` later.
-->

Lấy một ví dụ, ta bắt đầu bằng việc tạo một tập dữ liệu nhân tạo lưu trong file csv  `../data/house_tiny.csv` (csv - *comma-separated values - giá trị tách nhau bằng dấu phẩy*).
Dữ liệu trong các định dạng khác cũng có thể được xử lý tương tự.
Hàm `mkdir_if_not_exist` dưới đây để đảm bảo rằng thư mục `../data` có tồn tại.

> Chú thích `# Saved in the d2l package for later use` (*Lưu lại trong gói d2l để dùng sau*) là nhãn riêng cho các hàm, các lớp hoặc các lệnh import sau này được lưu trong gói `d2l` để ta có thể trực tiếp gọi hàm `d2l.mkdir_if_not_exist()` sau này.

Reviewer 2: ```suggestion
Chú thích `# Saved in the d2l package for later use` (*Lưu lại trong gói d2l để dùng sau*) là kí hiệu đánh dấu các hàm, lớp hoặc các lệnh import được lưu trong gói `d2l`, để sau này ta có thể trực tiếp gọi hàm `d2l.mkdir_if_not_exist()`.
```

Below we write the dataset row by row into a csv file.
-->

> Sau đây ta ghi tệp dữ liệu vào file csv theo kiểu hàng nối hàng.

Reviewer 2: ```suggestion
Sau đây ta ghi tệp dữ liệu vào file csv theo từng hàng một.
```
Mình đọc chữ `nối` sợ liên tưởng linh tinh thành `concat` các kiểu nên sửa lại.
